SOLID/donations.yml

```diff
@@ -0,0 +1,35 @@
+- ticker: SOLID
+
+  recipient: 
+    name: United Vision for Idaho
+    address: 1912 W Jefferson St, Boise, ID 83702 USA
+    tel: +1 208 869 3131
+    contact: uvidaho@gmail.com
+
+  ada: 100 # <- These must be numbers. Can be decimals though. See below
+  usd: 163 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1620431928 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots 
```

Comment:
Hmm... apparently the `yq` utility, which is being used by the CI to validate the file references, has a bug and returns the comments as well. Can you please remove the comments in lines 16-19?
Thanks